Shared/Features/Freaks/ViewModel/FreaksViewModel.swift

```diff
@@ -1,9 +1,27 @@
 import Foundation
 
-final class FreaksViewModel {
-    var freaks: [Freak] = [
-        Freak(imageName: "person.crop.square", firstName: "Vlad", lastName: "Stanescu"),
-        Freak(imageName: "person.crop.square", firstName: "Mihai", lastName: "Pantiru"),
-        Freak(imageName: "person.crop.square", firstName: "Raluca", lastName: "Ionescu"),
-    ]
+final class FreaksViewModel: ObservableObject {
+    @Published var freaks: [Freak] = load("MockData.json")
+}
+
+func load<T: Decodable>(_ filename: String) -> T {
```

Comment:
I think it's better to move this function to another file. It's a generic function that decodes any JSON file, not really related to  Freaks View Model, even tho we call it in the ViewModel.